gsadjust/gui/tabs/drift.py

```diff
@@ -814,7 +835,7 @@ def show_all_columns(delta_view):
         Helper function to reset columns in the delta_model
         :param delta_view: view to reset
         """
-        model = delta_view.model()
+        model = delta_view.model().sourceModel()

```

Comment:
When you have a proxy in place, you need to reach "through" the proxy to get to the actual model. An alternative is just to keep a reference to the model around ourselves (e.g. `self.delta_model`) -- but if that is replaced, we still need to hook into the proxy model for it to work.

When you have an value on an object e.g. `self.delta_model` and replace it by assigning to it `self.delta_model = MyNewModel()` that changes the object in `self.delta_model` but *all existing links* are still pointing at the old object.